src/main/java/org.jitsi.meet.test/mobile/WelcomePageView.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright @ Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.meet.test.mobile;
+
+import io.appium.java_client.*;
+import io.appium.java_client.pagefactory.*;
+
+import org.jitsi.meet.test.mobile.base.*;
+
+import org.openqa.selenium.*;
+
+/**
+ * Page object for the mobile welcome page.
+ */
+public class WelcomePageView extends AbstractMobilePage
+{
+    /**
+     * Conference room name input locator.
+     */
+    @AndroidFindBy(accessibility = "Input room name.")
+    @iOSFindBy(accessibility = "Input room name.")
```

Comment:
Since these 2 will almost always be the same, is there a way to combine them into one? Something like `@MobileFindBy` perhaps.

Reply:
I was trying to figure this out, but there's no way currently to do that as far as I know. I've tried asking the community, but no response so far:

https://discuss.appium.io/t/find-by-accessibility-id-on-both-android-and-ios-using-one-annotation/19746

Reply:
Haha, great minds think alike, I guess :-P Maybe we can at least make the string a static constant to avoid typos?